include/links-model.php

```diff
@@ -168,15 +169,16 @@ public function home_url( $language ) {
 	 *
 	 * @since 3.4
 	 *
-	 * @param array $default  Array of language default data values containing home and search URLs.
-	 * @param array $language Language data.
-	 * @return array Language data with URLs addeddefault.
+	 * @param array $additional_data Array of language additional data.
+	 * @param array $language        Language data.
+	 * @return array Language data with additional home and search URLs added.
```

Comment:
"add" twice in the sentence, I think we can remove `additional`:
```suggestion
	 * @return array Language data with home and search URLs added.
```

Reply:
Done in 514f8d5759de010583cea8cdafd9342065b8e740